R/tm_missing_data.R

```diff
@@ -79,18 +82,24 @@ tm_missing_data <- function(label = "Missing data",
   module(
     label,
     server = srv_page_missing_data,
-    server_args = list(plot_height = plot_height, plot_width = plot_width, ggplot2_args = ggplot2_args),
+    server_args = list(
+      parent_dataname = parent_dataname, plot_height = plot_height,
+      plot_width = plot_width, ggplot2_args = ggplot2_args
+    ),
     ui = ui_page_missing_data,
     filters = "all",
-    ui_args = list(pre_output = pre_output, post_output = post_output, ggtheme = ggtheme)
+    ui_args = list(
+      parent_dataname = parent_dataname, pre_output = pre_output,
+      post_output = post_output, ggtheme = ggtheme
+    )
   )
 }
 
-ui_page_missing_data <- function(id, data, pre_output = NULL, post_output = NULL, ggtheme) {
+ui_page_missing_data <- function(id, data, parent_dataname, pre_output = NULL, post_output = NULL, ggtheme) {
   ns <- NS(id)
   datanames <- names(data)
 
-  if_subject_plot <- !is.null(get_join_keys(data))
+  if_subject_plot <- length(parent_dataname) > 0 && parent_dataname %in% datanames
```

Comment:
If not in datanames should we print some warning?

Reply:
I don't think so, because in non-cdisc case we would also get warnings (the modules now have no way of knowing if the data coming in is cdisc or not - so I think this is the best we can do)